onnx_tf/backend.py

```diff
@@ -207,22 +214,25 @@ def _onnx_node_to_tensorflow_op(cls,
                                   node,
                                   tensor_dict,
                                   handlers=None,
-                                  opset=None):
+                                  opset=None,
+                                  strict=True):
     """
     Convert onnx node to tensorflow op.
 
     Args:
       node: Onnx node object.
       tensor_dict: Tensor dict of graph.
       opset: Opset version of the operator set. Default 0 means using latest version.
-
+      strict: whether to enforce semantic equivalence between the original model
+        and the converted tensorflow model, defaults to True (yes, enforce semantic equivalence).
+        Changing to False is strongly discouraged.
```

Comment:
Could we list up target op that strict would be applied to?

Reply:
Good idea, will do soon.

Reply:
@tjingrant And detail doc for these ops is more expected......